iothub/service/samples/PnpServiceSamples/noComponentSample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+
+namespace PnpServiceSample
+{
```

Comment:
Is this the name we concluded?

Reply:
We haven't concluded a name for service sample. We can always change this.

Reply:
currently changing to same as Java